punchcut/src/source/glyf.rs

```diff
@@ -11,33 +12,27 @@ pub use scaler::Scaler;
 
 pub use read_fonts::types::Point;
 
-use read_fonts::types::F26Dot6;
+use read_fonts::types::{F26Dot6, Fixed};
 
 /// Context for loading for TrueType glyphs.
-#[derive(Clone, Debug)]
+#[derive(Clone, Default, Debug)]
 pub struct Context {
     /// Unscaled points.
     unscaled: Vec<Point<i32>>,
     /// Original scaled points.
     original: Vec<Point<F26Dot6>>,
-    /// Storage for variation deltas.
-    deltas: Vec<Point<i32>>,
-}
-
-impl Default for Context {
-    fn default() -> Self {
-        Self::new()
-    }
+    /// Storage for simple glyph deltas.
+    deltas: Vec<Point<Fixed>>,
+    /// Storage for composite glyph deltas.
+    composite_deltas: Vec<Point<Fixed>>,
+    /// Temporary storage for computing interpolated deltas.
+    interp_points: Vec<Point<Fixed>>,
```

Comment:
nits:

1. all your other `Vec<Point<Fixed>>` are named deltas, why is this one points while commented to be deltas?
1. You've used `Vec<Point<Fixed>>` enough it might be worth considering a type alias, e.g. `simple_glyph_deltas: Deltas`

Reply:
`interp_points` gets passed as `working_points` to the internal delta computation code and it does actually store points that are used for interpolating deltas. I can change the name to match that and add a note to the comment about actual usage.

For the deltas, I'd feel better with a `type Delta = Point<Fixed>` to express the intent without throwing away the information that it's a `Vec`